project/Dependencies.scala

```diff
@@ -51,9 +51,7 @@ object Dependencies {
   val testDependencies =
     (mockito ++ specsBuild ++ junitInterface ++ assertj ++ awaitility ++ slf4jtest ++ logback).map(_ % Test)
 
-  val standaloneApiWSDependencies = javaxInject ++ sslConfigCore ++ akkaStreams.map(
-    _.exclude("com.typesafe", "*")
-  ) ++ testDependencies
```

Comment:
This exclude was introduced in #604 as a workaround for akka-http missing Scala 3 artifacts